hardhat.config.ts

```diff
@@ -174,6 +174,18 @@ const config: HardhatUserConfig = {
         mnemonic: process.env.MNEMONIC || "",
       },
     },
+    sepolia: {
+      url: process.env.RPC_URL || "https://rpc.notadegen.com/eth/sepolia",
+      chainId: 11155111,
+      live: true,
+      gasPrice: 20000000000, // 20 gwei
+    },
+    ethereum: {
+      url: process.env.ETHEREUM_ARCHIVE_NODE_URL || "https://eth-mainnet.public.blastapi.io",
+      chainId: 1,
+      live: true,
+      timeout: 1200000, // 20 minutes
+    },
```

Comment:
If we are going to add this support in this PR, let's follow this commit as a guideline: https://github.com/VenusProtocol/oracle/pull/124/commits/f2f5e7d94c3acc7cceebbcc739933808f353a936

Changes:
* .env.example file
* Configuration of the etherscan plugin in the hardhat config file, for sepolia and ethereum 
* Environment variables to use: `FORK`, `FORKED_NETWORK` and `ARCHIVE_NODE_${process.env.FORKED_NETWORK}` in the file `hardhat.config.ts` and in any place where we were using `FORK_??`. For example:
```
const FORK_MAINNET = process.env.FORK_MAINNET === "true";
```
should be replaced by:
```
const FORK_MAINNET = process.env.FORK === "true" && process.env.FORKED_NETWORK === "bscmainnet";
```

Reply:
Done https://github.com/VenusProtocol/isolated-pools/pull/324/commits/533b6df0de471b771604ebe3b68d2273a28068b7